### ISSUE TITLE
how to render a sample html message using read only mode

### ISSUE THREAD

**Issue by prathameshkurunkar7** [10play/10tap-editor#117]
### Questions:

* Is the way mentioned below how I'm supposed to build a sample renderer? I am unsure how to set the content using the useEditorBridge hook, I used initialContent as the suited prop. But is this how I am supposed to implement or is there something I am missing?
* Moreover I am mapping RichTextEditor for say a list of messages, this is currently making every message scrollable individually, is there any way to stop this?

### Code Sample Screenshot:

![image](https://uploads.linear.app/2729553a-1a7d-4564-8c8e-acf8c04dbbee/9f95d59a-67df-4828-943f-b3bf94f0edc4/4e602804-6cf5-4c8b-a3e0-3509bcad879a?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzI3Mjk1NTNhLTFhN2QtNDU2NC04YzhlLWFjZjhjMDRkYmJlZS85Zjk1ZDU5YS02N2RmLTQ4MjgtOTQzZi1iM2JmOTRmMGVkYzQvNGU2MDI4MDQtNmNmNS00YzhiLWEzZTAtMzUwOWJjYWQ4NzlhIiwiaWF0IjoxNzE2ODgzNzAyLCJleHAiOjMzMjg3NDQzNzAyfQ.GyAfK9fnRWebpH1UUWhWZhCGBd2ZFK6Fjjc1l17xkbU "=631x")

### Output Screenshot:

![image](https://uploads.linear.app/2729553a-1a7d-4564-8c8e-acf8c04dbbee/53714a9c-7c4b-4d61-90da-0b9cff1ed1b2/0b0dfc2b-caad-4185-9336-51a517d82425?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzI3Mjk1NTNhLTFhN2QtNDU2NC04YzhlLWFjZjhjMDRkYmJlZS81MzcxNGE5Yy03YzRiLTRkNjEtOTBkYS0wYjljZmYxZWQxYjIvMGIwZGZjMmItY2FhZC00MTg1LTkzMzYtNTFhNTE3ZDgyNDI1IiwiaWF0IjoxNzE2ODgzNzAyLCJleHAiOjMzMjg3NDQzNzAyfQ.pvAmpWP1dJxp7uAnfRVDFqLiRkKyy6aHMwm5RufWcDY "=423x")

**Comment by GuySerfaty** [comment]
hey @prathameshkurunkar7, Im not fully understand what you are asking for bullet 1, can you try to explain yourself better here? for the second one, In case you are not rendering the editor on the whole screen consider to `avoidIosKeyboard: false`

**Comment by prathameshkurunkar7** [comment]
@GuySerfaty apologies, let me rephrase - I want to build a Renderer using TenTap (basically a component which shows formatted html in read only mode) example - for showing formatted messages in a chat. My question was how am I supposed to provide my message content(text to be rendered) to the useEditorBridge hook? Is there any other way to do this than what I've done?

**Comment by GuySerfaty** [comment]
> @GuySerfaty apologies, let me rephrase - I want to build a Renderer using TenTap (basically a component which shows formatted html in read only mode) example - for showing formatted messages in a chat. My question was how am I supposed to provide my message content(text to be rendered) to the useEditorBridge hook? Is there any other way to do this than what I've done?

Oh I see, yeah it looks ok to me to use [initialContent](https://10play.github.io/10tap-editor/docs/api/useEditorBridge#initialcontent), it's a HTML string

**Comment by prathameshkurunkar7** [comment]
Thanks for the feedback!
